tests/functional/test_uninstall.py

```diff
@@ -411,6 +411,31 @@ def test_uninstall_from_reqs_file(script, tmpdir):
     )
 
 
+@pytest.mark.network
+def test_uninstall_per_setup_options(script):
+    """
+    Check that uninstall works with per-setup options are specified
+    """
+    script.scratch_path.joinpath("test-req.txt").write_text(
+        textwrap.dedent("""
+            six --install-option="-v"
+        """)
+    )
```

Comment:
Why is `six` significant here? Can this not be reproduced with a locally available test package?

Reply:
I guess a locally available package is OK as well, should I change it?

Reply:
@uranusjr fixed